chat/public/css/chat.css

```diff
@@ -223,6 +223,10 @@ body.view-in-course .course-wrapper.chromeless {
     width: 80%;
 }
 
+.chat-block .response-button button:disabled {
+    opacity: 0.5;
```

Comment:
@xitij2000 Sorry, I mislead you in my previous comment. Now that we reversed opacity settings, I think we can probably just set `opacity: 1` here? It's already `1` in the Studio, but we want to override the `0.5` value in the LMS.